msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -3007,8 +3029,14 @@ private PrivateMessage processPvtMsgReplyAllSendHelper(boolean preview, Boolean
 	  return rrepMsg;
   }
 
-
-
+  private void manageTagAssociation(Long msgId) {
+    log.debug("msgId " + msgId + " - selectedTags " + selectedTags);
+    if (msgId != null && ServerConfigurationService.getBoolean("tagservice.enable.integrations", false) && isInstructor() && selectedTags != null) {
```

Comment:
This has to be enabled by default.

Is this property documented in default.sakai.properties? 

tagservice.enable.integrations

Reply:
I think we should also add S2U-28 to the documentation, was S2U-28 enabled by default?

Reply:
Ok, Yes, I changed it to enable it by default